### DISCUSSION TITLE
release of v1

### DISCUSSION THREAD

**Discussion by MCjat0** [muxinc/next-video#185] [General]
Hi guys do you already know when release for v1 is planned? I would really love to have the customizable player available with this library.

Thanks!

**Comment by luwes** [comment]
thanks for the feedback and question!

we're working towards a v1 release and most things are in place I believe, we need some more test coverage and a few features.

it's already possible to use a custom player, have you tried following the steps here 
https://github.com/muxinc/next-video#custom-player

cheers!

**Reply by MCjat0** [reply]
I have seen this part but this only allows me to switch to a custom player and not modify the default player right?

Another question, is it possible to not create the video folder when using the init command during setup?
